cadctl/cmd/cluster-missing/cluster-missing.go

```diff
@@ -38,6 +41,35 @@ var ClusterMissingCmd = &cobra.Command{
 			return fmt.Errorf("could not start client: %w", err)
 		}
 
+		// ---------------------------------------
+		// will be fetched from PD
+		clusterID := "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+		// requires credentials to be at a specific dir (~/.ocm.json)
+		ocmClient, err := ocm.NewOcmClient("")
```

Comment:
maybe rename the function to `NewOcmClientFromPath`
so it's clear the "" is a filepath